### PR TITLE
Update "Apply Unique Value Renderer" sample

### PR DESCRIPTION
**Pull request by changanxian**
- Added comments in code and tried matching the rendering color with the common design
- Added screenshot
- Updated metadata
- Added README

### CODE REVIEW

File: lib/samples/apply_unique_value_renderer/apply_unique_value_renderer.png
Comment (kmueller-gis) [inline lib/samples/apply_unique_value_renderer/apply_unique_value_renderer.png:1]: I don't believe any of the other screenshots contain the app bar.

File: lib/samples/apply_unique_value_renderer/README.metadata.json [hunk 1]
```diff
@@ -2,10 +2,21 @@
     "category": "Visualization",
     "description": "Render features in a layer using a distinct symbol for each unique attribute value.",
     "ignore": false,
-    "images": [],
-    "keywords": [],
+    "images": ["apply_unique_value_renderer.png"],
+    "keywords": [
+        "draw", "renderer", "symbol", "symbology", "values"
```
Comment (kmueller-gis) [inline lib/samples/apply_unique_value_renderer/README.metadata.json:7]: keywords should contain the relavant_apis too.

You can actually delete this file and use the script `tools/readme_scripts/metadata_checker.py` to generate a new one based on the README.md

File: lib/samples/apply_unique_value_renderer/apply_unique_value_renderer_sample.dart [hunk 2]
```diff
@@ -27,54 +27,62 @@ class ApplyUniqueValueRendererSample extends StatefulWidget {
 
 class _ApplyUniqueValueRendererSampleState
     extends State<ApplyUniqueValueRendererSample> {
+  // create a controller for the map view.
   final _mapViewController = ArcGISMapView.createController();
 
   @override
   void initState() {
```
Comment (kmueller-gis) [inline lib/samples/apply_unique_value_renderer/apply_unique_value_renderer_sample.dart:34]: An `onMapViewReady` function should be added and the `initState` code should be moved there.

File: lib/samples/apply_unique_value_renderer/apply_unique_value_renderer_sample.dart [hunk 3]
```diff
@@ -27,54 +27,62 @@ class ApplyUniqueValueRendererSample extends StatefulWidget {
...
     map.operationalLayers.add(featureLayer);
     _mapViewController.arcGISMap = map;
   }
 
   @override
   Widget build(BuildContext context) {
     return Scaffold(
+      // add a map view to the widget tree and set a controller.
       body: ArcGISMapView(
         controllerProvider: () => _mapViewController,
```
Comment (kmueller-gis) [inline lib/samples/apply_unique_value_renderer/apply_unique_value_renderer_sample.dart:61]: Add `onMapViewReady`

File: lib/samples/apply_unique_value_renderer/apply_unique_value_renderer_sample.dart [hunk 4]
```diff
@@ -27,54 +27,62 @@ class ApplyUniqueValueRendererSample extends StatefulWidget {
...
+  /// Configure a unique value renderer.
   Renderer? _configureUniqueValueRenderer() {
     final stateOutlineSymbol = SimpleLineSymbol(
         style: SimpleLineSymbolStyle.solid, color: Colors.white, width: 0.7);
 
+    // create fill symbols for each region.
     final pacificFillSymbol = SimpleFillSymbol(
         style: SimpleFillSymbolStyle.solid,
-        color: Colors.blue,
+        color: const Color.fromARGB(255, 0, 0, 255),
```
Comment (kmueller-gis) [inline lib/samples/apply_unique_value_renderer/apply_unique_value_renderer_sample.dart:74]: why the change?

File: lib/samples/apply_unique_value_renderer/README.metadata.json (hunk 1 above)
Comment (changanxian) [reply lib/samples/apply_unique_value_renderer/README.metadata.json:7]: actually, it is `create_metadata_from_README.py -s` to create the file

Comment (kmueller-gis) [reply lib/samples/apply_unique_value_renderer/README.metadata.json:7]: oops... my bad.

File: lib/samples/apply_unique_value_renderer/apply_unique_value_renderer_sample.dart (hunk 4 above)
Comment (changanxian) [reply lib/samples/apply_unique_value_renderer/apply_unique_value_renderer_sample.dart:74]: The design uses a pure blue.  'Colors.blue' is a shade blue not to match the design color.

**Review by kmueller-gis — APPROVED** [review]
